### ISSUE TITLE
Consolidate, review and improve nmdc-schema's issue->branch->PR->release policy

### ISSUE THREAD

**Issue by turbomam** [microbiomedata/nmdc-schema#1045]
Especially intended to ensure that stakeholders know when a nmdc-schema merge or release will put new constraints on the data. We want to avoid repeating the recent case in which I merged new classes in support of NEON but also included stricter CURIe rules for `alternative_identifiers` 

We already have some docuemtnation about this in files checked into the repo, but those should at least be listed out if not edited. I will list them in this issue soon.

cc @aclum @mslarae13 @cmungall @sujaypatil96 @pkalita-lbl ... please tag or reach out to others that you want to contribute

example PR that could be merged but hasn't been assessed for its impact:
- https://github.com/microbiomedata/nmdc-schema/pull/1001

**Comment by mslarae13** [comment]
I SUPPORT THIS ISSUE!

Also, with this, developing a schedule for release and migrations!

**Comment by shreddd** [comment]
@turbomam - can you outline the proposed policy so we can make sure we are on the same page? 
(Eventually this should be an ADR)

**Comment by turbomam** [comment]
> I SUPPORT THIS ISSUE!
> 
> Also, with this, developing a schedule for release and migrations!

I support that too. Could scheduling be a separate issue please? I guess the release/migration dimension should be included in this issue though.

I see this issue as about 
- making sure that the issue->branch->PR->release workflow is documented so anyone can follow it
- making sure that we have policies about what can be contained in a PR merge or release, and what stakeholders should be consulted in what ways

**Comment by turbomam** [comment]
> @turbomam - can you outline the proposed policy so we can make sure we are on the same page? (Eventually this should be an ADR)

Yes, but probably not until Tuesday the 18th.

**Comment by aclum** [comment]
This seems pretty under control now with the monthly release process, closing.
